src/aspects/metadata.ts

```diff
@@ -0,0 +1,31 @@
+import type { IAspect, Stack } from "aws-cdk-lib";
+import type { IConstruct } from "constructs";
+import { LibraryInfo, MetadataKeys } from "../constants";
+
+export type GuConstruct = {
```

Comment:
TODO should we use symbols here?

Reply:
And also, should we make it clearer that this is about the *public* API of @guardian/cdk in the naming somehow?

Reply:
Should we just use the class name here and save some lines of code.